tests/test_io.py

```diff
@@ -0,0 +1,66 @@
+"""Test the trajectories io module."""
+import numpy as np
+from mrinufft.trajectories.io import read_trajectory, write_trajectory
+from mrinufft.trajectories.trajectory2D import initialize_2D_radial
+from mrinufft.trajectories.trajectory3D import initialize_3D_cones
+from pytest_cases import parametrize_with_cases
+
+
+class CasesIO:
+    """Cases 2 for IO tests, each has different parameters."""
+
+    def case_trajectory_2D(self):
+        """Test the 2D trajectory."""
+        trajectory = initialize_2D_radial(
+            Nc=32,
+            Ns=256,
+            tilt="uniform",
+            in_out=False
+        ).astype(np.float32)
```

Comment:
Should we not also test also for float64 ?

Reply:
Umm no, not supported

Reply:
in this case, we need to determine the behavior (and test it) should the data be cast prior to saving (with a warning to the user ...) or raise an error ? 
This should also be mentionned in the reference doc.